docs/backend-api/backend-api-installation.md

```diff
@@ -0,0 +1,30 @@
+# Backend API Installation
+
+Require `shopsys/backend-api` package in the `composer.json` or run
+
+```bash
+composer require shopsys/backend-api ^8.0
```

Comment:
hmm this is new notation. Till now we always use exact version like `composer require shopsys/backend-api:7.2.2`. But i like you solution more. But for consistency reasons.....

Reply:
hmm i see why in composer.patch is no shopsys/backend-api...but still it is reversed

Reply:
I found only one occurence of `composer require` in our documentation and it is

> you can do this by `composer require helios-ag/fm-elfinder-bundle:^9.2 --update-with-dependencies

with the caret ("roof") annotation...

So what should be there?

Reply:
in README.md of backed api is just `composer require shopsys/backend-api`

Reply:
changed to `composer require shopsys/backend-api`